client/shipping-label/views/purchase/steps/packages/package-info.js

```diff
@@ -68,11 +68,29 @@ const PackageInfo = ( { packageId, selected, all, unpacked, dimensionUnit, weigh
 			return ( <div className="wcc-package__desc"><span className="wcc-package__desc-name">Package - </span><span className="wcc-package__desc-dimensions">{ renderPackageDimensions( pckg, dimensionUnit ) }</span></div> );
 		}
 
-		const packageOptions = _.map( all, renderPackageOption );
+		const groups = _.reduce( flatRateGroups, ( result, groupTitle, groupId ) => {
+			result[ groupId ] = { title: groupTitle, definitions: {} };
+			return result;
+		}, {
+			custom: { title: __( 'Custom Packages' ), definitions: {} },
+		} );
+
+		_.forEach( all, ( groupPckg, groupPckgId ) => {
+			const group = groupPckg.group_id ? groups[ groupPckg.group_id ] : groups.custom;
+			group.definitions[ groupPckgId ] = groupPckg;
```

Comment:
Consider using `_.filter()` to build group definitions, and bring it into the above `_.reduce()` call. You should be able to determine group packages by: `_.filter( all, { group_id: groupId } )`.